infracost-usage-example.yml

```diff
@@ -469,6 +469,11 @@ resource_usage:
   #
   azurerm_app_service_environment.my_service:
      operating_system: linux # Override the operating system of the instance, can be: linux, windows.
+
+  azurerm_databricks_workspace.my_workspace:
+    monthly_all_purpose_compute_dbu: 500 # Monthly number of All-purpose Compute Databricks Units (DBUs).
```

Comment:
@alikhajeh1  I'm wondering if these should `..._dbu_hours`.  The azure pricing pages talks about DBUs in DBU-month, DBU-hour, DBU-sec.  

At a minimum we should update the comment to say that it is DBU-hour

Reply:
good point, all three can be changed to be line this (we mention _hrs in the contributing doc for usage file params):
```
monthly_all_purpose_compute_dbu_hrs: 500 # Monthly number of All-purpose Compute Databricks Units in DBU-hours.
```